tests/test_server.py

```diff
@@ -0,0 +1,53 @@
+import json
+import os.path
+import unittest
+
+class PyGeoAPIServerTests(unittest.TestCase):
+
+    def setUp(self):
+        # Import current pygeoapi Flask app module
+        from pygeoapi import flask_app
+        self.client = flask_app.APP.test_client()
+
+    def tearDown(self):
+        pass
+
+    def testExecuteProcessMetadata(self):
+        response = self.client.get(f'/processes/execute')
+        self.assertEqual(response.status_code, 200)
+
+    def testExecuteProcessExecution(self):
+        path = os.path.abspath(os.path.join(
+            os.path.dirname(__file__), 'test_data/carbon_willamette.invs.json'))
```

Comment:
Since we repeat `os.path.dirname(__file__)` a few times here, would it be worth abstracting to the module level?  I suppose the same could be said about the whole willamette json file.  What do you think?